starknet_py/net/account/account_client.py

```diff
@@ -43,22 +31,15 @@ class AccountClient(Client):
     def __init__(
         self,
         address: AddressRepresentation,
-        key_pair: KeyPair,
         net: Network,
+        signer: BaseSigner,
```

Comment:
I think we can accept either a signer o keypair that will be used with the default signer.

Reply:
I think we may need to get rid of `create_account` in format that we use currently.

Reply:
How would we replace it? I think it's good to have a easy way of deploying an account through starknet.py.

Reply:
I guess it can stay for now, but it can't accept signer - it can only deploy the default account. Still, I would accept signer | keypair in account's constructor.

Reply:
Okay, we can do `signer | keypair`